pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/util/SystemExecutables.java

```diff
@@ -0,0 +1,45 @@
+package com.linkedin.gradle.python.util;
+
+import org.apache.tools.ant.taskdefs.condition.Os;
+
+/**
+ * Enum that allows for platform independent resolution of executables
+ */
+public enum SystemExecutables {
```

Comment:
Thats certainly doable.  The only question I haven't answered yet is how many executables are we talking.  5?  Sure, we can do that.  500? Bad design.  I'll keep things as we are now, once we get closer to having this merged in and everything is functional, we can revisit it.  I will most likely take your advice tho, if there's more than a dozen exe's, i'd be shocked.